addons/mail/static/src/components/message_reaction_per_emoji/message_reaction_per_emoji.scss

```diff
@@ -0,0 +1,43 @@
+// ------------------------------------------------------------------
+// Layout
+// ------------------------------------------------------------------
+
+.o_MessageReactionPerEmoji {
+    border-radius: $o-mail-rounded-rectangle-border-radius-sm;
+    cursor: pointer;
+    display: flex;
+    margin-right: 2px;
+    width: fit-content;
+}
+
+.o_MessageReactionPerEmoji_count {
+    margin: 5px;
+}
+
+.o_MessageReactionPerEmoji_emojiUnicode {
+    padding: 5px;
+}
+
+// ------------------------------------------------------------------
+// Style
+// ------------------------------------------------------------------
+
+.o_MessageReactionPerEmoji {
+    background: lightgrey;
+    border-color: lightgrey;
+    border-style: solid;
+    border-width: thin;
+}
+
+.o_MessageReactionPerEmoji_count {
+    color: white;
+}
+
+.o-active {
+    border-color: blue;
+}
+
+.o-activeCount {
+    color: blue;
+    font-weight: bold;
+}
```

Comment:
These are modifiers on root and `count` part, so they should be nested rule on these part, not defined globally.